packages/react-core/src/components/TextInputGroup/examples/TextInputGroupFilters.tsx

```diff
@@ -52,13 +58,13 @@ export const TextInputGroupFilters: React.FunctionComponent = () => {
   return (
     <TextInputGroup>
       <TextInputGroupMain icon={showSearchIcon && <SearchIcon />} value={inputValue} onChange={handleInputChange}>
-        <ChipGroup>
+        <LabelGroup>
           {currentChips.map((currentChip) => (
-            <Chip key={currentChip} onClick={() => deleteChip(currentChip)}>
+            <Label key={currentChip} variant="outline" onClose={() => deleteChip(currentChip)}>
```

Comment:
It's because internally, Chips are Labels with the outline variant. We do have a codemod issue opened to cover converting the deprecated Chip to Label (though I believe the goal to at minimum update the Chip to the deprecated path).

Reply:
Maybe we need to update the path and also add the variant.  Or at least a warning.